mlir/lib/Dialect/Quant/Transforms/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_mlir_dialect_library(MLIRQuantTransforms
+  LowerQuantOps.cpp
+  StripFuncQuantTypes.cpp
+
+  ADDITIONAL_HEADER_DIRS
+  {$MLIR_MAIN_INCLUDE_DIR}/mlir/Dialect/Quant/Transforms
+
+  DEPENDS
+  MLIRQuantTransformsIncGen
+
+  LINK_LIBS PUBLIC
```

Comment:
Given the dialect dependencies of the `lower-quant-ops` pass, you probably need link-time dependencies on the corresponding dialects.